deploy/charts/harvester/values.yaml

```diff
@@ -338,16 +338,16 @@ harvester-network-controller:
     repository: rancher/harvester-network-controller
     pullPolicy: IfNotPresent
     # Overrides the image tag whose default is the chart appVersion.
-    tag: v0.3.0-rc1
+    tag: master-head
```

Comment:
note: we change the tag to master-head in the `master` branch and will use the correct version like `v0.3.0-rc2` in the release branch.